src/styles/globals.css

```
@@ -10,6 +10,38 @@ body {
+
+  [data-testid='settings-panel'] input:not([type]):focus,
+  [data-testid='settings-panel'] input[type='text']:focus,
+  [data-testid='settings-panel'] input[type='password']:focus,
+  [data-testid='settings-panel'] input[type='number']:focus,
+  [data-testid='settings-panel'] input[type='url']:focus,
+  [data-testid='settings-panel'] input[type='email']:focus,
+  [data-testid='settings-panel'] textarea:focus,
+  [data-testid='settings-panel'] select:focus {
+    @apply !border-primary;
+    box-shadow: 0 0 0 2px
+      color-mix(in srgb, var(--color-primary) 20%, transparent);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)